modules/cnv-pxe-booting-with-mac-address.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-configuring-pxe-booting.adoc
+
+[id="cnv-pxe-booting-with-mac-address-{context}"]
+= PXE booting with a specified MAC address
+
+PXE booting, or network booting, is supported in {ProductName}.
+Network booting allows a computer to boot and load an
+operating system or other program without requiring a locally attached
+storage device. For example, you can use it to choose your desired OS
+image from a PXE server when deploying a new host.
+
+.Prerequisites
+
+* A bridge must be configured on all nodes. The bridge must be connected to
+an interface that allows PXE booting.
+
+.Procedure
+
+. Configure a PXE network on the cluster:
+
+.. Create the NetworkAttachmentDefinition file for PXE network `pxe-net-conf`:
++
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: pxe-net-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "type": "cnv-bridge",
+      "bridge": "br1"
+    }'
+----
++
+[NOTE]
+====
+In this example, the VMI will be attached through a trunk port
+to the bridge `<br1>`.
+====
+
+. Edit the virtual machine instance configuration file to include the
+details of the interface and network.
+
+.. Specify the network and MAC address, if required by the PXE server.
+If the MAC address is not specified, a value is assigned
+automatically. However, note that at this time, MAC addresses
+assigned automatically are not persistent.
++
+Ensure that `bootOrder` is set to `1` so that the interface boots first.
+In this example, the interface is connected to a network called
+`<pxe-net>`:
++
+----
+interfaces:
+- masquerade: {}
+  name: default
+- bridge: {}
+  name: pxe-net
+  macAddress: de:00:00:00:00:de
```

Comment:
the pxe-net CRD use the bridge plugin that doesn't support for mac address you need to chain it with the cnv-tuning plugin like this

```yaml
{
      "cniVersion": "0.3.1",
      "name": "bridge-tuning",
      "plugins": [
        {
          "type": "cnv-bridge",
          "bridge": "br-tuning",
          "ipam": {}
        },
        {
          "type": "cnv-tuning"
        }
      ]
    }
```

Reply:
Oh wow, thanks for the heads up. Is this documented upstream anywhere? I can't find it. 

Please let me know if the updated version is correct.

Reply:
Hi you can take a look in this PR https://github.com/kubevirt/user-guide/pull/220

Reply:
Thanks @SchSeba ! Can you confirm that this looks good to you?

`apiVersion: "k8s.cni.cncf.io/v1"
kind: NetworkAttachmentDefinition
metadata:
  name: pxe-net-conf
spec:
  config: '{
      "cniVersion": "0.3.1",
      "name": "pxe-net-conf",
      "plugins": [
        {
          "type": "cnv-bridge",
          "bridge": "br1",
          "ipam": {}
        },
        {
          "type": "cnv-tuning" <1>
        }
      ]
    }'`

Reply:
looks good :)